configs/manifests/telliot.yml

```diff
@@ -45,11 +45,16 @@ spec:
             secretKeyRef:
               name: telliot-main
               key: NODE_URL
-        - name: ETH_PRIVATE_KEY
+        - name: NODE_WEBSOCKET_URL
```

Comment:
need to also update the example.env file

Reply:
Ok, fixed in other places as well.